### ISSUE TITLE
`output` Plug-in Attribute

### ISSUE THREAD

**Issue by mottosso** [pyblish/pyblish-qml#153]
### Goal

Prove user feedback on what a particular plug-in outputs, e.g. an .abc or .mb file.

### Implementation

```python
class MyPlugin(...)
  output = [".mb", ".abc"]
```

**Comment by tokejepsen** [comment]
Any motivation for this?

**Comment by mottosso** [comment]
Yeah, have a look at the comment from mkolar [here](https://github.com/pyblish/pyblish-qml/issues/80#issuecomment-98692553).
